.github/workflows/deploy-docs.yml

```diff
@@ -18,10 +18,10 @@ jobs:
     - name: Checkout
       uses: actions/checkout@v4
 
-    - name: Use Node.js LTS 18.x
+    - name: Use Node.js LTS 20.x
       uses: actions/setup-node@v4.0.2
       with:
-        node-version: 18.x
+        node-version: 20.x
```

Comment:
For the other repos we use `20.11.0`
Should that matter here?

Reply:
In the other repositories we want to make sure that we always _test_ with the **minimal supported version** of Node.js, which is 20.11.0 and 21.2.0.

In this particular workflow, we don't test our modules but build the documentation. Using the latest LTS version seems more reasonable here, so that we can benefit from bug fixes and improvements.